geocoder_tester/world/france/iledefrance/test_addresses.csv

```diff
@@ -512,9 +512,6 @@ query,lon,lat,limit,expected_name,expected_housenumber,expected_street,expected_
 12BIS Chemin d'Orveau Vayres-sur-Essonne,,,,12BIS Chemin d'Orveau,,,,91820
 12BIS Chemin d'Orveau 91820,,,,12BIS Chemin d'Orveau,,,Vayres-sur-Essonne,
 12BIS Chemin d'Orveau,2.351197,48.438203,,12BIS Chemin d'Orveau,,,,91820
-3 Avenue Lyautey Vert-le-Petit,,,,3 Avenue Lyautey,,,,91710

```

Comment:
If we are not sure about what is expected when geocoding this query, I think the test does not produce any value. That's why I delete this one instead of trying to correct it.